src/components/SearchSection/SearchSection.js

```diff
@@ -1,26 +1,85 @@
 import React, { useState } from 'react';
 import './SearchSection.css';
 import SearchBar from '../SearchBar/SearchBar';
+import SearchResults from '../SearchResults/SearchResults';
+import resources from '../../../src/data/resources.js';
 
 const SearchSection = () => {
     const [searchBarText, setSearchBarText] = useState("");
+    const [suggestedItem, setSuggestedItem] = useState([]);
+    const [showResults, setShowResults] = useState(false);
 
     const handleSearchInput = onChange => {
-        setSearchBarText(onChange.target.value);
```

Comment:
This way of searching is fine for now, but in the future we should implement something called "debouncing." A guide (courtesy of ACM Hack lol) can be found [here](https://hack.uclaacm.com/posts/fall2019/js-chats-3/#application-power-of-hooks-with-debouncing).